.jenkins/libcxx_tests.Jenkinsfile

```diff
@@ -101,44 +25,27 @@ def ACClibcxxTest(String label, String compiler, String build_type) {
     }
 }
 
-def cleanup(){
-    node("nonSGX") {
-        timeout(GLOBAL_TIMEOUT_MINUTES) {
-            cleanWs()
-            checkout scm
-            oe.deleteRG([XENIAL_RG, BIONIC_RG])
-        }
-    }
-}
-
 try {
     oe.emailJobStatus('STARTED')
-    for (int i = 1 ; i <= AGENT_NUM ; i++ ) {
-        parallel "Deploy Ubuntu 16.04 #${i}" :  { ACCDeployVM("${XENIAL_LABEL}-${i}".toLowerCase(), "xenial" , "eastus", XENIAL_RG, "${VHD_URL_XENIAL}") },
-                 "Deploy Ubuntu 18.04 #${i}" :  { ACCDeployVM("${BIONIC_LABEL}-${i}".toLowerCase(), "bionic", "westeurope", BIONIC_RG, "${VHD_URL_BIONIC}") }
-    }
 
-    registerJenkinsSlaves()
-
-    parallel "libcxx ACC1604 clang-7 Debug" :          { ACClibcxxTest(XENIAL_LABEL, 'clang-7', 'Debug') },
-             "libcxx ACC1604 clang-7 Release" :        { ACClibcxxTest(XENIAL_LABEL, 'clang-7','Release') },
-             "libcxx ACC1604 clang-7 RelWithDebInfo" : { ACClibcxxTest(XENIAL_LABEL, 'clang-7', 'RelWithDebinfo') },
-             "libcxx ACC1604 gcc Debug" :              { ACClibcxxTest(XENIAL_LABEL, 'gcc', 'Debug') },
-             "libcxx ACC1604 gcc Release" :            { ACClibcxxTest(XENIAL_LABEL, 'gcc', 'Release') },
-             "libcxx ACC1604 gcc RelWithDebInfo" :     { ACClibcxxTest(XENIAL_LABEL, 'gcc', 'RelWithDebInfo') },
-             "libcxx ACC1804 clang-7 Debug" :          { ACClibcxxTest(BIONIC_LABEL, 'clang-7', 'Debug') },
-             "libcxx ACC1804 clang-7 Release" :        { ACClibcxxTest(BIONIC_LABEL, 'clang-7', 'Release') },
-             "libcxx ACC1804 clang-7 RelWithDebInfo" : { ACClibcxxTest(BIONIC_LABEL, 'clang-7', 'RelWithDebinfo') },
-             "libcxx ACC1804 gcc Debug" :              { ACClibcxxTest(BIONIC_LABEL, 'gcc', 'Debug') },
-             "libcxx ACC1804 gcc Release" :            { ACClibcxxTest(BIONIC_LABEL, 'gcc', 'Release') },
-             "libcxx ACC1804 gcc RelWithDebInfo" :     { ACClibcxxTest(BIONIC_LABEL, 'gcc', 'RelWithDebinfo') }
+    parallel "libcxx ACC1604 clang-7 Debug" :          { ACClibcxxTest("LIBCXX-1604", 'clang-7', 'Debug') },
```

Comment:
What you could do is: 

Change the original from 

XENIAL_LABEL = "LIBCXX-${BUILD_NUMBER}-1604"
BIONIC_LABEL = "LIBCXX-${BUILD_NUMBER}-1804"

-> 

XENIAL_LABEL = "LIBCXX-1604"
BIONIC_LABEL = "LIBCXX-1804"

Then this section does not need to be changed and there's less pasting of the same variable. Which I think is preferrable.

Reply:
I agree with this